app/src/main/java/com/kickstarter/ui/activities/CampaignDetailsActivity.kt

```diff
@@ -30,14 +32,16 @@ class CampaignDetailsActivity : BaseActivity<CampaignDetailsViewModel.ViewModel>
         this.viewModel.outputs.pledgeContainerIsVisible()
             .observeOn(AndroidSchedulers.mainThread())
             .compose(bindToLifecycle())
-            .subscribe { ViewUtils.setGone(campaign_details_pledge_container, !it) }
+            .subscribe {
+                binding.campaignDetailsPledgeContainer.isGone = !it
+            }
 
         this.viewModel.outputs.url()
             .observeOn(AndroidSchedulers.mainThread())
             .compose(bindToLifecycle())
-            .subscribe { web_view.loadUrl(it) }
+            .subscribe { binding.webView.loadUrl(it) }
```

Comment:
The campaign details has stop loading, attache video comparing latest master behaviour VS this branch behaviour

https://user-images.githubusercontent.com/4083656/128578693-db532681-fa9b-45c8-88de-6ca172fe4978.mp4

Reply:
Fixed
https://user-images.githubusercontent.com/1075310/128724440-1844eef3-e812-4b62-b026-7cf2ecde1490.mp4